go/kubebuilder/.generate/gen-go-sample.sh

```diff
@@ -0,0 +1,265 @@
+    corev1 "k8s.io/api/core/v1" \
+    "k8s.io/apimachinery/pkg/api/errors" \
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"' $FILENAME
+
+    sed -i '/apimachinery\/pkg\/runtime/ a\
+    "k8s.io/apimachinery/pkg/types"' $FILENAME
+
+    sed -i '/'"$MEMCACHE_CONTROLLER_OLD_RECONCILER"'/,$c\'"$MEMCACHE_CONTROLLER_NEW_RECONCILER" $FILENAME
+}
+
+function update_memcached_webhook() {
+    FILENAME=api/v1alpha1/memcached_webhook.go
+
+    # Update imports
+    sed -i '/import (/ a\
+    "errors"' $FILENAME
+
+    # Updating validation logic
+    sed -i 's/\/\/ TODO(user): fill in your defaulting logic./ if r.Spec.Size == 0 { \
+		r.Spec.Size = 3 \
+	}/' $FILENAME
+
+    sed -i '/return nil/d' $FILENAME
+
+    sed -i 's/\/\/ TODO(user): fill in your validation logic upon object creation./ return validateOdd(r.Spec.Size)/' $FILENAME
+
+    sed -i 's/\/\/ TODO(user): fill in your validation logic upon object update./ return validateOdd(r.Spec.Size)/' $FILENAME
+
+    sed -i 's/\/\/ TODO(user): fill in your validation logic upon object deletion./ return nil/' $FILENAME
+
+
+    sed -i -e '$afunc validateOdd(n int32) error { \
+	    if n%2 == 0 { \
+		    return errors.New("Cluster size must be an odd number") \
+	    } \
+	    return nil \
+    }' $FILENAME
+}
+
+function update_default_kustomization() {
+    FILENAME=config/default/kustomization.yaml 
+
+    # Uncommenting lines with [WEBHOOK] prefix in 
+    # config/default/kustomization.yaml to enable Webhooks
+    sed -i '45,70 s/^#//' $FILENAME
+    sed -i '21 s/^#//' $FILENAME
+    sed -i '23 s/^#//' $FILENAME
+    sed -i '35 s/^#//' $FILENAME
+    sed -i '40 s/^#//' $FILENAME
+}
+
+function update_cache_v1alpha1_memcached_sample() {
+    sed -i 's/foo: bar/size: 3/' config/samples/cache_v1alpha1_memcached.yaml
+}
+
+function scaffold_go_project() {
+
+    header_text "Removing the existing example project"
+    rm -rf memcached-operator
+
+    header_text "Creating project directory"
+    mkdir memcached-operator
+    cd memcached-operator
+
+    header_text "Initializing new project and creating API"
+    operator-sdk init --repo github.com/example/memcached-operator --domain example.com
+    operator-sdk create api --group cache --version v1alpha1 --kind Memcached --controller --resource
+
+    # Update memcached_types.go
+    update_memcached_types
+
+    # Update memcached_controllers.go
+    update_memcached_controllers
+
+    header_text "Updating the generated code for specific resource"
+    make generate
+
+    header_text "Generating manifests"
+    make manifests
+
+    header_text "Scaffolding webhook"
+    operator-sdk create webhook --group cache --version v1alpha1 --kind Memcached --defaulting --programmatic-validation
+
+    # Update memcached_webhook.go
+    update_memcached_webhook
+
+    # Update config/default/kustomization.yaml
+    update_default_kustomization
+
+    # Update config/samples/cache_v1alpha1_memcached.yaml
+    update_cache_v1alpha1_memcached_sample
+}
```

Comment:
we need also: 
- replace the target test with: https://sdk.operatorframework.io/docs/building-operators/golang/quickstart/#configuring-your-test-environment
- run make docker-build IMG=quay.io/example/memcached-operator:v0.0.1
- Update the Readme for the IMGE be `quay.io/example/memcached-operator:v0.0.1` instead of `quay.io/example-inc/memcached-operator:v0.0.1`
- Then run: https://github.com/operator-framework/operator-sdk-samples/pull/139/files#diff-7a48155790f7a965e68bffccaefa0465R61-R70

Reply:
Hi @varshaprasad96, 

Shows that it still missing the bundle steps. See:

```
  header_text "bulding the project ..."
  make docker-build IMG=$operIMG

  header_text "integrating with OLM ..."
  sed -i".bak" -E -e 's/operator-sdk generate kustomize manifests/operator-sdk generate kustomize manifests --interactive=false/g' Makefile; rm -f Makefile.bak

  header_text "generating bundle and building the image $bundleIMG ..."
  make bundle IMG=$operIMG
  make bundle-build BUNDLE_IMG=$bundleIMG
```

Also, currently, we generate the package manifest as well.  So, it means add in the Makefile: 

```
  header_text "adding packagemanifests target to the Makefile ..."
  cat ../packagemanifests-target.txt >> Makefile
```

```

# Options for "packagemanifests".
ifneq ($(origin CHANNEL), undefined)
PKG_CHANNELS := --channel=$(CHANNEL)
endif
ifeq ($(IS_CHANNEL_DEFAULT), 1)
PKG_IS_DEFAULT_CHANNEL := --default-channel
endif
PKG_MAN_OPTS ?= $(PKG_CHANNELS) $(PKG_IS_DEFAULT_CHANNEL)

# Generate package manifests.
packagemanifests: kustomize manifests
	operator-sdk generate kustomize manifests -q --interactive=false
	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
	$(KUSTOMIZE) build config/manifests | operator-sdk generate packagemanifests -q --version $(VERSION) $(PKG_MAN_OPTS)
```

And the call `make packagemanifests`